core/src/main/kotlin/org/evomaster/core/problem/rest/classifier/ModelAccuracy.kt

```diff
@@ -1,18 +1,25 @@
 package org.evomaster.core.problem.rest.classifier
 
+import org.evomaster.core.problem.rest.data.RestCallResult
+
 interface ModelAccuracy {
 
     /**
      * Estimate the accuracy of the model
      */
     fun estimateAccuracy() : Double
 
+    /**
+     * Estimate the precision of the model
+     */
+    fun estimatePrecision400(): Double
```

Comment:
if we provide both measures, we need to expand the comments to clarify what is the difference between accuracy and precision here